static/js/util/courseList.js

```diff
@@ -84,7 +87,10 @@ export function makeCourseStatusDisplay(course: Object, now: moment = moment()):
     } else {
       if (firstRun.course_id) {
         let courseInfoUrl = `${SETTINGS.edx_base_url}/courses/${firstRun.course_id}/about`;
-        return <Button bsStyle="success" href={courseInfoUrl} target="_blank">ENROLL</Button>;
+        return <Button bsStyle="success" href={courseInfoUrl} target="_blank">
+          ENROLL
+          <span className="sr-only"> in {firstRun.title}</span>
+        </Button>;
```

Comment:
It looks like the extra comment tags are only visible in the inspector on my local machine, I don't see it on micromasters-ci for example